lib/scripts/repo_export_in_batches.rb

```diff
@@ -9,7 +9,6 @@
 
 # set classes to have records exported and classes that will do the actual conversion
 generic_file_converter = {model_class: GenericFile, converter_class: InvenioRdmRecordConverter}
-page_converter = {model_class: Page, converter_class: InvenioRdmRecordConverter}
 collection_converter = {model_class: "", converter_class: ""}
 # TODO: Implement collection conversion
 converters = [generic_file_converter, page_converter]
```

Comment:
`page_converter` should be removed here too, right?

Reply:
Ah, good catch.